hedyorg/hedy
# 💻 create super teacher

Fixes #5443
Fixes #5422 
Fixes #5474
Fixes #5371
Fixes #5451 

**How to test?**
First you have to make your teacher account a super teacher. Do so by:
- logging in as an admin,
- go to users overview,
- then mark your account as super teacher.
> Test this works expectedly!

Then log in with your super teacher account, now you see `Super teacher` in the nav bar. With this you can:
- view feedback submitted by users,
   - if there's no feedback, create one so that you could see it.
   - you should see then all info related to it. 
   - you can also click on create issue, which takes opens a new tab with a github issue filled with the selected feedback information.
- view public adventures
   - and delete any adventure
- view support teacher page
   - connect one teacher to another  (this feature isn't significant for now)
   - what's important is to get a view of teachers who indicated they needed support
   - 

**DB changes**
- added a `is_super_teacher (0 or 1)` field to the user table.
